dashboard/app/assets/javascripts/small_footer.js.jsx

```diff
@@ -179,7 +179,7 @@ window.dashboard.footer = (function () {
               }}/>
               <small>
                 {this.renderCopyright()}
-                <a className="more-link" href="#"
+                <a className="more-link" href="javascript:void(0)"
```

Comment:
hah. I'd gone back and forth on using # vs. javascript:void(0). I thought I saw SO answer saying # was better, but it's been problematic in various ways. I wonder if we should be doing a broader cleanup of these.